templates/backOffice/default/RewriteUrl/module-configuration.html

```diff
@@ -25,106 +25,128 @@
 
         {hook name="module.configuration" location="module_configuration" modulecode="$module_code"}
 
-        <div class="general-block-decorator">
-            <div class="row">
-                <div class="col-md-12">
-                    <div class="panel panel-default">
-                        <div class="panel-body">
-                            <div>
-                                <label>{intl l="Enable URL rewriting"}</label>
-                                <div class="make-switch switch-small js_switch_rewrite_url" data-on="success" data-off="danger" data-on-label="<i class='glyphicon glyphicon-ok'></i>" data-off-label="<i class='glyphicon glyphicon-remove'></i>">
-                                    <input type="checkbox" {if $isRewritingEnabled}checked="checked"{/if}>
-                                </div>
-                            </div>
-                            <span class="help-block">{intl l="Set the config variable 'rewriting_enable'"}</span>
+
+        <div class="row">
+            <div class="panel panel-default col-md-4">
+                <div class="panel-body">
+                    <div>
+                        <label>{intl l="Enable URL rewriting"}</label>
+                        <div class="make-switch switch-small js_switch_rewrite_url" data-on="success" data-off="danger" data-on-label="<i class='glyphicon glyphicon-ok'></i>" data-off-label="<i class='glyphicon glyphicon-remove'></i>">
+                            <input type="checkbox" {if $isRewritingEnabled}checked="checked"{/if}>
                         </div>
                     </div>
+                    <span class="help-block">{intl l="Set the config variable 'rewriting_enable'"}</span>
+                </div>
+            </div>
+        
+            <!-- index.php redirection switch -->
```

Comment:
Dans le cas du html, le commentaire est en effet une bonne idée, ça aide à se repérer.  
par contre pour la fermeture de la div, il est préférable de donner plus d'info
exemple : 
`<!-- end of index.php redirection switch -->`
à la place de 
`<!-- ################################-->`

Pour info, tu peux aussi faire des commentaires en smarty qui ne seront pas visibles après la compilation du code, c'est des commentaires visibles uniquement par les devs.  
```smarty
{* A Smarty comment *}
```